docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst

```diff
@@ -59,6 +59,26 @@ The following code snippets are shell scripts that can be used as orientation ho
 **See** https://download.bareos.org/bareos/release/ **for applicable releases and distributions.**
 See :ref:`section-UniversalLinuxClient` if your distribution is not present.
 
+To simplify the, all Linux distribution and FreeBSD repositories on https://download.bareos.org/ and https://download.bareos.com/ contain a script named :file:`add_bareos_repositories.sh`.
+
+Download the :file:`add_bareos_repositories.sh` script
+matching the requested Bareos release
+and the distribution of the target system.
+
+Copy the script onto the target system and execute it with a shell (:command:`sh`) as root (e.g. using :command:`sudo`),
+or manually perform the steps that are documented in the script.
+
+For example the script URL for bareos-21 and Debian 11 is:
+
+* https://download.bareos.org/bareos/release/21/Debian_11/add_bareos_repositories.sh
+* or for Bareos subscription customers:
+
+   * https://download.bareos.com/bareos/release/21/Debian_11/add_bareos_repositories.sh
+   * .. note::
+
+        To download :file:`add_bareos_repositories.sh` you must authenticate against https://download.bareos.com. If this is inconvenient, you can alternatively download :file:`add_bareos_repository_template.sh` and replace ``BAREOS_USERNAME`` and ``BAREOS_PASSWORD`` in there manually.
+
```

Comment:
This is a copy and paste of the beginning of debian installation. Maybe we should add more value here or remove the redundant information in the debian section.
 
Don't we want to modify also the different OS ?

Reply:
This PR is still a draft and not yet finished. I suggest to postpone the PR, until it is done.